node/src/chain_spec.rs

```diff
@@ -152,24 +170,69 @@ pub fn local_testnet_config() -> Result<ChainSpec, String> {
 /// Configure initial storage state for FRAME modules.
 fn testnet_genesis(
     wasm_binary: &[u8],
-    initial_authorities: Vec<(AuraId, GrandpaId)>,
+    initial_authorities: Vec<AuthorityKeys>,
     root_key: AccountId,
     endowed_accounts: Vec<AccountId>,
-    endowed_utxos: Vec<sr25519::Public>,
     _enable_println: bool,
-) -> GenesisConfig {
+) -> GenesisConfig
+{
+    const ENDOWMENT: u128 = 1 << 80;
+    // minimum balance set in the runtime. check `lib.rs` of runtime module.
+    const STASH: u128 = 40_000;
+
     // only Alice contains 400 million coins.
-    let genesis = endowed_utxos
+    let (genesis_utxos, locked_genesis) = initial_authorities
         .first()
         .map(|x| {
-            // may need to create a const variable to represent 1_000 and 100_000_000
-            pallet_utxo::TransactionOutput::new_pubkey(
-                1_000 * 100_000_000 * 400_000_000 as pallet_utxo::Value,
-                H256::from_slice(x.as_slice()),
-            )
+            let x_h256 = H256::from(x.controller.clone());
+            let x_stash_h256= H256::from(x.stash.clone());
+
+            let num_of_utxos = 5;
+            let value = (400_000_000 * MLT_UNIT) / num_of_utxos;
```

Comment:
I feel like this magic number would be nicer if it was defined elsewhere.

ORIG_SUPPLY or something